src/compositor/seat/seat.c

```diff
@@ -114,8 +114,10 @@ seat_handle_key(struct wlc_seat *seat, const struct wlc_input_event *ev)
       return;
 
    wlc_keyboard_update_modifiers(&seat->keyboard, ev->device);
+   struct wlc_modifiers mods = {0, 0};
```

Comment:
I assume this is used intentionally instead of seat->keyboard.modifiers to avoid translation caused by CTRL + ALT to some other key symbol?